CHANGELOG.adoc

```diff
@@ -312,6 +312,8 @@ The only way to export diagram as SVG image is to use the ReactFlow toolbar loca
 - https://github.com/eclipse-sirius/sirius-web/issues/2689[#2689] [diagram] Fix the borderNode moves that is no longer anchored to its parent.
 - https://github.com/eclipse-sirius/sirius-web/issues/2682[#2682] [diagram] Fix an error in diagram selection that can lead to a node remaining selected in the diagram but not in the explorer.
 - https://github.com/eclipse-sirius/sirius-web/issues/2605[#2605] [diagram] Fix an issue that prevents last node or edge to remain selected if the new selection is not displayed on the diagram.
+- https://github.com/eclipse-sirius/sirius-web/issues/2919[#2919] [diagram] Reset node position if drag and drop fails.
+ the diagram.
```

Comment:
Typo? Not sure where "the diagram" line comes from.

Reply:
Done